src/app/join-us/page.tsx

```diff
@@ -1,32 +1,151 @@
+import Faq from '@/components/faq/faq';
+import { Box, Divider, Link, Paper, Typography } from '@mui/material';
 import Container from '@mui/material/Container';
+import Image from 'next/image';
 import React from 'react';
 import './join-us.scss';
 
 export default function JoinUs() {
   return (
     <Container maxWidth="lg" className="container">
-      <Typography className="join-us-header" variant="h3">
-        Join Us
-      </Typography>
-      <Typography variant="body2" paragraph>
-        Unlock the power of your skills and expertise to drive positive change
-        in British Columbia through OpenBC. At OpenBC, we&apos;re passionate
-        about leveraging your unique abilities for the advancement of our
-        province. Whether you&apos;re a tech enthusiast, a creative visionary,
-        or a community leader, your skills can play a crucial role in addressing
-        challenges and creating opportunities.
-      </Typography>
-      <Typography variant="body2" paragraph>
-        Ready to contribute to British Columbia through Open source projects?
-        Join us at OpenBC and become a part of our active and welcoming
-        community. Here&apos;s how you can engage:
-      </Typography>
-      <Typography variant="body2">
-        <strong>
-          Email openbc.ca@gmail.com for opportunities to contribute.
-        </strong>
+      <Box className="sub-container-2-col">
+        <Box className="box-size-50">
+          <Image
+            src="https://source.unsplash.com/random?wallpapers"
+            alt="Description"
+            width={500}
+            height={500}
+            style={{ width: '100%' }}
+          />
+        </Box>
+        <Box className="box-size-50__marginX">
+          <Typography className="sub-header" variant="h2">
+            Our Mission
+          </Typography>
+          <Typography variant="body2" paragraph>
+            At OpenBC, we&apos;re passionate about leveraging your unique
+            abilities for the advancement of our province.
+          </Typography>
+          <Typography variant="body2" paragraph>
+            <strong>
+              Contribute to Open Source Projects for BC, from Anywhere, with
+              OpenBC.
+            </strong>
+          </Typography>
+        </Box>
+      </Box>
+      <Box className="margin-left-4 margin-bottom-4">
+        <Typography className="sub-header" variant="h5">
+          Visions
+        </Typography>
+        <Typography variant="body2" paragraph>
+          Quis neque, eu et ipsum amet, vel et. Varius integer enim pellentesque
+          ornare pharetra faucibs arcu. Mauris blandit egestas nibh. eu et ipsum
+          amet, vel et. Varius integer enim pellentesque ornare pharetra faucibs
+          arcu. Mauris blandit egestas nibh.
+        </Typography>
+      </Box>
+      <Box className="box-size-40__center">
+        <Box className="display-flex__space-between">
```

Comment:
I like how this was cleaned up to the SCSS, but in my last comment, I recommended `space-around` to more closely match the Design:

https://github.com/OpenBCca/openbc-web/pull/90#discussion_r1477135146

However, if `space-between` is the actual desired look, then we can leave it!